modules/programs/eww.nix

```
@@ -45,26 +96,60 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
-    xdg.configFile."eww".source = cfg.configDir;
-
-    programs.bash.initExtra = mkIf cfg.enableBashIntegration ''
-      if [[ $TERM != "dumb" ]]; then
-        eval "$(${ewwCmd} shell-completions --shell bash)"
-      fi
-    '';
-
-    programs.zsh.initExtra = mkIf cfg.enableZshIntegration ''
-      if [[ $TERM != "dumb" ]]; then
-        eval "$(${ewwCmd} shell-completions --shell zsh)"
-      fi
-    '';
-
-    programs.fish.interactiveShellInit = mkIf cfg.enableFishIntegration ''
-      if test "$TERM" != "dumb"
-        eval "$(${ewwCmd} shell-completions --shell fish)"
-      end
-    '';
-  };
+  config = mkIf cfg.enable (mkMerge [
+    {
+      assertions = [
+          {
+            assertion = !(cfg.scssConfig == null && cfg.configDir == null) or !(cfg.yuckConfig == null && cfg.configDir == null);
+            message = "You can have either yuck & scss or configDir enabled"
```

```suggestion
            assertion = !(cfg.scssConfig == null && cfg.configDir == null) || !(cfg.yuckConfig == null && cfg.configDir == null);
            message = "You can have either yuck & scss or configDir enabled";
```
Seems like a language mishap.

---

Yeah I am aware, but I have been so busy focusing other things than this pr.